Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -829,4 +829,42 @@ theorem secondCountable_of_separable [SeparableSpace α] : SecondCountableTopolo
     exact hUV (ball_subset_of_comp_subset (hk hxy) hUU' (hk hz))
 #align uniform_space.second_countable_of_separable UniformSpace.secondCountable_of_separable
 
+section DiscreteUniformity
+
+/-- A Cauchy filter in a discrete uniform space is contained in a principal filter-/
+theorem Cauchy_Discrete_le_principal {X : Type _} {uX : UniformSpace X}
+    (hX : uniformity X = 𝓟 idRel) {α : Filter X} (hα : Cauchy α) : ∃ x : X, α ≤ 𝓟 {x} := by
```

Comment:
All those `𝓟 idRel` should be spelled bot. And the conclusion should be an equality. Is there any reason why you don’t make `uX` an instance parameter?

Reply:
I have changed the 𝓟 idRel to ⊥ and the ≤ to equality. The reason why I do not want uX to be an instance parameter is that one might want to apply the lemma in the setting where a type has no uniformity at all, and then along the proof put on it the discrete one. If you want I can create two lemmas, one with an instance parameter and the other one like this, with a usual assumption.